paper-menu-button.html

```diff
@@ -268,6 +270,14 @@
               }
             }];
           }
+        },
+
+        /**
+         * This is the element intended to be bound as the focus target
+         * for the `iron-dropdown` contained by `paper-menu-button`.
+         */
+        _dropdownContent: {
```

Comment:
Because in this case, it has nothing to do with `focusTarget`, it is just incidentally the thing that we want to bind to `focusTarget` in `iron-dropdown`.